src/rules/__tests__/require-tothrow-message.test.js

```diff
@@ -1,13 +1,13 @@
 import { RuleTester } from 'eslint';
```

Comment:
can you rename the test file as well?

Reply:
I renamed the file. Not sure why this isn't showing "outdated"?

Reply:
the comment itself is not (since the line I commented on is the same), even though the file has been renamed.

(at least that's my guess 😛)